ansible/roles/bastion-lite/tasks/main.yml

```diff
@@ -12,8 +12,14 @@
     - copy_env_private_key
 
 - name: Generate host .ssh/config Template
-  become: no
-  local_action: template src={{ role_path }}/templates/bastion_ssh_config.j2 dest={{output_dir}}/ssh-config-{{ env_type }}-{{ guid }}
+  become: false
+  local_action: >-
+    template src={{
+    role_path
+    }}/templates/bastion_ssh_config.j2 dest={{
+    output_dir
+    }}/ssh-config-{{
+    env_type }}-{{ guid }}
```

Comment:
This should use the other syntax, inlining is not recommended in our Contributing guidelines

Something like:

```yaml
- name: Generate host .ssh/config Template
  become: false
  template:
    src: "{{ role_path }}/templates/bastion_ssh_config.j2"
    dest: "{{ output_dir }}/ssh-config-{{ env_type }}-{{ guid }}"
  delegate_to: localhost
```

(to be tested)